### PR TITLE
affiliation normalization: use function from inspire-utils

### PR DESCRIPTION
**Pull request by MJedr**
cern-sis/issues-inspire#48

### CODE REVIEW

File: inspirehep/modules/workflows/tasks/actions.py [hunk 1]
```diff
@@ -1079,37 +1016,33 @@ def _assign_institution_reference_to_affiliations(author_affiliations, already_m
                 already_matched_affiliations_refs[complete_affiliation['value']] = complete_affiliation['record']
 
 
-def normalize_affiliations(obj, eng):
-    matched_affiliations = {}
-    for author in obj.data.get("authors", []):
+def normalize_author_affiliations(obj, eng):
+    search = Search(index="records-hep", using=current_search_client)
```
Comment (drjova) [inline inspirehep/modules/workflows/tasks/actions.py:1020]: do wee need to create a new search instance and hardcode the index?

File: docker-compose.yml [hunk 2]
```diff
@@ -107,6 +108,7 @@ services:
     extends:
       file: services.yml
       service: indexer
+    platform: linux/amd64
```
[the remark this answers is not in the document]
Comment (MJedr) [reply docker-compose.yml:111]: cause otherwise it doesn't work on my laptop

File: inspirehep/modules/workflows/tasks/actions.py (hunk 1 above)
Comment (MJedr) [reply inspirehep/modules/workflows/tasks/actions.py:1020]: it was like that before, I can use LiteratureSearch if that works on next

File: docker-compose.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (drjova) [reply docker-compose.yml]: We have to find a different solution, can we generate an override for m1?

[the remark this answers is not in the document]
Comment (MJedr) [reply docker-compose.yml]: I found another config flag for docker which is exporting that setting, it's ok for now

**Review by drjova — APPROVED** [review]

File: inspirehep/modules/workflows/tasks/actions.py (hunk 1 above)
Comment (MJedr) [reply inspirehep/modules/workflows/tasks/actions.py]: It's ok with `LiteratureSearch`, I would keep it

**Pull request merged**